tests/cypress/tests-e2e/drawing.cy.js

```diff
@@ -284,6 +302,17 @@ describe('Drawing module tests', () => {
                     new RegExp(`<LabelStyle><color>${KML_STYLE_BLACK}</color></LabelStyle>`),
                 ])
             })
+
+            cy.get('[data-cy="drawing-style-delete-button"]').click({ force: true })
+            cy.clickDrawingTool(EditableFeatureTypes.ANNOTATION)
+            cy.get('[data-cy="ol-map"]:visible').click()
+            readCoordinateClipboard('feature-style-edit-coordinate-copy')
+            cy.log('Coordinates for marker can be copied in drawing mode')
```

Comment:
The log should be place before the test take place otherwise if the test fails we don't see it